pkg/virt-controller/watch/vm/vm.go

```diff
@@ -3262,3 +3273,146 @@ func hasEphemeralHotplugVolumes(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMa
 
 	return false
 }
+
+func (c *Controller) patchHotplugVolumes(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) error {
+	if !vmi.IsRunning() {
+		return nil
```

Comment:
Is this correct? I mean if we don't return any error but there isn't been any update, I don't think the reconciliation loop is triggered again once the VMI is running

Reply:
VM reconciliation happens whenever VMI status changes so if VMI goes from Pending->Running for example the VM controller will reconcile again